context.go

```diff
@@ -379,6 +379,12 @@ func (c Context) Interface(key string, i interface{}) Context {
 	return c
 }
 
+// Any is a wrapper around Context.Interface.
+func (c Context) Any(key string, i interface{}) Context {
+	c.l.context = enc.AppendInterface(enc.AppendKey(c.l.context, key), i)
```

Comment:
Comment is saying that this is a wrapper around `Context.Interface` but you copied its implementation and you are not calling into `Context.Interface`? I think you can just call into `Context.Interface`? Compiler will probably inline this anyway.

Reply:
Sorry, fixed it